spokestack/nlu/result.py

```diff
@@ -0,0 +1,34 @@
+"""
+This module contains the Result class for the NLU.
+"""
+from typing import Any, Dict
+
+
+class Result:
+    """ Convenience wrapper for NLU Results"""
+
+    def __init__(self, **kwargs) -> None:
```

Comment:
I would pass these parameters explicitly for documentation and typing

Reply:
ok I believe I have this the right way now. I also added documentation for each argument.